tests/PHPUnit/System/expected/test_twoVisitsWithCustomVariables_segmentMatchNONE__CustomVariables.getCustomVariables_day.xml

```diff
@@ -1,7 +1,279 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <results>
 	<result idSite="1">
-		<result date="2010-01-03" />
+		<result date="2010-01-03">
```

Comment:
I think you might need to modify https://github.com/piwik/piwik/blob/2.16.0-b1/tests/PHPUnit/System/TwoVisitsWithCustomVariablesSegmentMatchNONETest.php#L58-L77 and maybe add a special case so it doesn't match anything?